cmd/clusterctl/cmd/init.go

```diff
@@ -108,6 +109,8 @@ func init() {
 		"Wait for providers to be installed.")
 	initCmd.Flags().IntVar(&initOpts.waitProviderTimeout, "wait-provider-timeout", 5*60,
 		"Wait timeout per provider installation in seconds. This value is ignored if --wait-providers is false")
+	initCmd.Flags().BoolVar(&initOpts.ignoreValidationErrors, "ignore-validation-errors", false,
```

Comment:
Would it be better to have this flage as `--validate` to mirror the kubectl flag (and flip the direction of the bool)?

Reply:
Amended. I've kept the library option the same though.